tests/fixtures/sc-resolve-public-client-id.yaml

```diff
@@ -0,0 +1,436 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Connection:
+      - close
+      Host:
+      - soundcloud.com
+      User-Agent:
+      - Python-urllib/3.9
+    method: GET
+    uri: https://soundcloud.com/
+  response:
+    body:
+      string: "<script crossorigin src=\"https://a-v2.sndcdn.com/assets/49-5f48c9f5.js\"\
```

Comment:
Response body strings are trimmed wrt original for brevity. Otherwise leading to +2mb .yaml files